Added Documentation on Estimation of Equivalent Usage Emissions

IamLRBA opened the pull request:
As per issue #733 
cc
@benoit-cty 
I Added a new section in the `methodology.rst.txt` file explaining equivalent usage emissions.

benoit-cty commented on docs/_sources/methodology.rst.txt:
```diff
@@ -148,3 +148,42 @@ CodeCarbon uses a scheduler that, by default, calls for a measure every 15 secon
+Car Usage
+---------
+- **Emission factor**: *0.12 kgCO₂ per kilometer driven*.
+- This value is derived from the average emissions of a passenger car under normal driving conditions.
```
```suggestion
- This value is derived from the average emissions of a European passenger car under normal driving conditions.
```

benoit-cty commented on docs/_sources/methodology.rst.txt:
```diff
@@ -148,3 +148,42 @@ CodeCarbon uses a scheduler that, by default, calls for a measure every 15 secon
+
+Source
+------
+The emission factors used are defined in the `CodeCarbon source code <https://github.com/mlco2/codecarbon/blob/b83cc2a38f21e410f9137e3a98a36f0ccfb4c905/webapp/src/helpers/constants.ts>`_. They are based on publicly available data and general assumptions.
```
```suggestion
The emission factors used are defined in the `CodeCarbon source code <https://github.com/mlco2/codecarbon/blob/master/webapp/src/helpers/constants.ts>`_. They are based on publicly available data and general assumptions.
```
So it is kept up-to-date.

benoit-cty commented on docs/_sources/methodology.rst.txt:
```diff
@@ -148,3 +148,42 @@ CodeCarbon uses a scheduler that, by default, calls for a measure every 15 secon
+
+TV Usage
+--------
+- **Emission factor**: *0.084 kgCO₂ per hour of usage*.
```
Well, for TV we don't use emissions but only the average electricity used per day, based on a 6.5 hours of use :
Average TV consumption : (187_000 Wh_per_year / (365*24)) * 6.5 h_per_day = 138 Wh

That way, it's not an equivalent of emissions, only of consumption. I thought it's more understandable that way. But we could discuss this.

IamLRBA replied:
Thank you @benoit-cty  for explaining your approach—it makes sense that focusing on average electricity consumption per day  better simplifies things for the audience. Using consumption rather than emissions could indeed make the concept easier to grasp, especially for those less familiar with emissions calculations.

That said, i just think it might be worth considering if adding a reference to emissions alongside consumption (perhaps as an optional detail or in parentheses) could add context for those interested in the environmental impact as i am sure there are few of those as well. 

Regardless, I'm happy to discuss further and find a balance that works best!

benoit-cty reviewed (changes requested):
Thanks, I'm always happy to see new contributor !

I suggest some changes and can you add the US citizen impact ?

IamLRBA reviewed (commented):
Thanks for the corrections @benoit-cty

IamLRBA reviewed (commented):
@benoit-cty these are the changes as per your review. If there are any more corrections you'd need just let me know.
Thanks!

benoit-cty commented:
Thanks, can you add the calculus for the US Citizen ?

We based our calculus on an average annual emission of  13.3 Tons of CO2.eq for a US citizen. And we divide it per 52 weeks per year.

IamLRBA commented:
> Thanks, can you add the calculus for the US Citizen ?
> 
> We based our calculus on an average annual emission of 13.3 Tons of CO2.eq for a US citizen. And we divide it per 52 weeks per year.

Thank you for the feedback @benoit-cty ! 

### PR Update: Added Calculations for US Citizen Weekly Emissions  

I've made the following updates to address your request:  

1. **Added Calculations for US Citizen Weekly Emissions:**  
   - Based on the average annual emissions of 13.3 tons CO₂ equivalent for a US citizen, 
   - I calculated weekly emissions as: 
    \[
     \text{Weekly Emissions} = \frac{13.3}{52} \approx 0.256 \, \text{tons CO₂ equivalent per week.}
     \]
   - I incorporated this into the documentation, providing the formula and examples for clarity.  

2. **I updated Documentation Structure:**  
   - Added a new section titled **"US Citizen Weekly Emissions"** to present the information.  
   - I then clarified how total emissions can be compared to this benchmark.  

The revised documentation now includes:  
- A breakdown of equivalent usage comparisons for cars, TVs, and US Citizen emissions.  
- Clear formulas and examples for each comparison.  

Please let me know if there's anything else you'd like me to adjust! 🚀

IamLRBA commented:
@benoit-cty in this new [commit](https://github.com/mlco2/codecarbon/pull/749/commits/0b2548910a6df0bfc595f6d84dcb33df41d952d8) i'm aiming to solve issue #751

benoit-cty commented:
I really appreciate your contribution, but please do a separate PR for RAM. Because we can't change something like that in a minor version as it will impact people making comparison between run over time.  Changing RAM value must be in a V3 of CodeCarbon.

IamLRBA commented:
> I really appreciate your contribution, but please do a separate PR for RAM. Because we can't change something like that in a minor version as it will impact people making comparison between run over time. Changing RAM value must be in a V3 of CodeCarbon.

THanks for your reply @benoit-cty. I will do that.

benoit-cty commented:
Hello,
You change `_sources/methodology.rst.txt`, but for Sphinx it is the `docs/edit/methodology.rst` that need to be change and Sphinx create the `txt` and then the `html`.

You can generate the doc locally with `hatch run docs:build` (See https://github.com/mlco2/codecarbon/blob/master/CONTRIBUTING.md#installation )

benoit-cty commented:
Hi @IamLRBA I have rebased and merged your contribution, thank you !

I've removed the RAM part because it has been done differently in https://github.com/mlco2/codecarbon/pull/804

IamLRBA commented:
> Hi @IamLRBA I have rebased and merged your contribution, thank you !
> 
> I've removed the RAM part because it has been done differently in #804

Alright Thank you @benoit-cty, If there is any other issue you'd particularly like me to work on I'll be pleased to indulge.
